sound/smooth_swing_v2.h

```diff
@@ -103,6 +108,12 @@ class SmoothSwingV2 : public SaberBasePassThrough {
         state_ = SwingState::ON;
         
       case SwingState::ON:
+        //check for AccentSwingThreshold, presence of accent swings and if the accent player is stopped (this prevents clipping)
```

Comment:
This line is too long.
Also, "clipping" is when the sound is too loud to fit in - 32768 - 32768.
I think you mean "prevents cutting off the sound"
Also, we don't actually check if the accent player is stopped here anymore

Reply:
ok, will update.

Reply:
ok, moved all the math to hybrid_font.h.  It's pretty specialized for accent swings, but we could change it for other purposes down the road as required.